### PR TITLE
Created a Project Overview for iD Editor - the OSM web editor

### PR DESCRIPTION
**Pull request by camerons**
Currently links to a Quickstart that doesn't exist. But otherwise should be good.

### CODE REVIEW

File: en/metrics.rst [hunk 1]
```diff
@@ -150,6 +149,19 @@ Geospatial Libraries
   <script type="text/javascript" src="https://www.openhub.net/p/liblas/widgets/project_factoids_stats?format=js"></script>
   <script type="text/javascript" src="https://www.openhub.net/p/python-iris/widgets/project_factoids_stats?format=js"></script>
 
+Coordinate Reference System Libraries
```
Comment (kalxas) [inline en/metrics.rst]: Do we really want to list all the MetaCRS projects here?
Also, why is this commit in the iD pull request?

Comment (camerons) [inline en/metrics.rst]: @kalxas, it looks like the tweak I put into metrics docs got pulled in here.
Agree, we should now only reference proj4 from the MetaCRS list now. There are a number of projects which should be getting removed from metrics.rst now that we have decided on the final list for version 11.

Comment (kalxas) [inline en/metrics.rst]: @camerons if you want to revert the part that was pulled in, please create another pull request.

**Comment by cvvergara** [comment]
@camerons 
The pull request has errors when running the
`make small`

On line 17 of ideditor_overview.rst, change this:
```
:doc:`Open Street Map <http://www.openstreetmap.org>` editor
```
to this
```
`Open Street Map <http://www.openstreetmap.org>`_ editor
```

Change 
```
 :doc:`JOSM<jsom_overview>
```
to 
```
:doc:`JOSM <josm_overview>`
```

In overview.rst, you created references to files that don't exist in the [quickstarts](https://github.com/OSGeo/OSGeoLive-doc/tree/master/en/quickstart):
ideditor_quickstart.rst
josm_quickstart.rst

* :doc:`ideditor_overview` - [:doc:`QuickStart <../quickstart/ideditor_quickstart>`] - OpenStreetMap Web Editor
* :doc:`josm_overview` - [:doc:`QuickStart <../quickstart/josm_quickstart>`] - OpenStreetMap Desktop Editor

**Comment by camerons** [comment]
@cvvergara thanks for the review, ideditor broken links  are fixed.
@kalxas, in metrics.rst, I'm not sure what I've done. proj4 is still referenced in my prior request, I think it might have been referenced twice before. Anyway, another pull request is required later to remove the old metacrs projects.

**Pull request merged**